app/server/src/main/scala/org/bitcoins/server/BitcoinSServerMain.scala

```diff
@@ -207,6 +212,19 @@ class BitcoinSServerMain(override val serverArgParser: ServerArgParser)(implicit
   }
 
   def startBitcoindBackend(): Future[Unit] = {
+
+    val bitcoindF = for {
+      client <- bitcoindRpcConf.clientF
```

Comment:
`bitcoindRpcConf` doesn't seem initialized here.

Reply:
This is done here: 

https://github.com/Christewart/bitcoin-s-core/blob/2b4d8a659439b7f376ca97602319c14236a0e594/app/server/src/main/scala/org/bitcoins/server/BitcoinSServerMain.scala#L59